src/viur/core/skeleton.py

```diff
@@ -1760,8 +1760,7 @@ class RefSkel(RelSkel):
     @classmethod
     def fromSkel(cls, kindName: str, *args: list[str]) -> t.Type[RefSkel]:
         """
-            Creates a relSkel from a skeleton-class using only the bones explicitly named
-            in \*args
+            Creates a relSkel from a skeleton-class using only the bones explicitly named in args.
```

Comment:
```
            :return: A new instance of RefSkel
```
does not match with `RelSkel` :thinking: 
```suggestion
            Creates a ``RefSkel`` from a skeleton-class using only the bones explicitly named in args.
```